readme.md

```diff
@@ -103,6 +103,10 @@ Projects & resources building with iroh!
 
 - :fire: [Iroh Discord](https://iroh.computer/discord) - an open community dedicated to Iroh and its services, where you can to socialize and help other web and app developers from around the world.
 
+### Semi-official
+
+-  [@n0iroh](https://mastodon.social/@n0iroh)
+- [matrix room](https://matrix.to/#/!KRxGKHTZtCEENtOPRH:matrix.org?via=matrix.org&via=codelutin.com&via=mauve.moe)
```

Comment:
Does anyone of us hang out in there?

If not, perhaps we should not mention it.

Reply:
> Does anyone of us hang out in there?
> 
> If not, perhaps we should not mention it.

If you don't promote the space, t is a self-fulfilling prophecy that noone is there.

While I can udnerstand that iroh wants to be on Discord as well, I think it is crucial for the project to also welcome people who do not want to, or [cannot](https://toot.teckids.org/@nik/114448664731293416), agree to Discords terms of use.

For a project that in its very essence wants to foster decentralisation, having at least one decentralised communication platform promoted on their website to collaborate on the project should not be too hard to go for.

Reply:
Sure, I just want to avoid a situation where someone joins the channel from our recommendation without a team member being there.

Thank you for your additional data point on this. This reason is precisely why we initially created that channel.

Reply:
I don't really think that a team member being present should be a requirement for a space.  I don't have anything against listing purely community spaces.